test/models/csv_export_test.rb

```diff
@@ -0,0 +1,60 @@
+require_relative '../test_helper'
+
+describe CsvExport do
+  let(:user) { users(:deployer) }
+
+  describe "new CsvExport" do
```

Comment:
`describe "#pending!"` ... the exporter being new does not really make these tests any different